src/pretix/control/templates/pretixcontrol/orders/index.html

```diff
@@ -117,6 +117,9 @@ <h1>{% trans "Orders" %}</h1>
                                     {{ o.code }}
                                 </a>
```

Comment:
Perhaps we can move the `</a>` to the previous line to get rid of the trailing whitespace which onHover gets converted into a line connecting to the TEST MODE-label.

![screenshot from 2019-02-19 13-13-34](https://user-images.githubusercontent.com/654548/53014256-40be1700-3448-11e9-9f2c-8ffbc61917b0.png)